src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static void initialiseWelcomeMessage() {
+        String logo
+                = "******************************************************\n"
+                + "*                 Systems: [Online]                  *\n"
+                + "*                Protocol: [Dominion]                *\n"
+                + "*                    Race: [Terran]                  *\n"
+                + "******************************************************\n";
```

Comment:
I like how you followed the coding standards and split long lines into shorter lines. I see this in other places as well.